.evergreen/scripts/compile-libmongocrypt.sh

```diff
@@ -5,6 +5,11 @@ compile_libmongocrypt() {
   declare -r mongoc_dir="${2:?}"
   declare -r install_dir="${3:?}"
 
+  # When the target of the `--branch` option is updated, also update the value
+  # of `LIBMONGOCRYPT_GITREF` in .evergreen/scripts/kms-divergence-check.sh and
+  # then run that script to ensure that there is no divergence (or resolve any
+  # observed divergence by syncing the KMS sources from libmongocrypt into this
+  # project)
```

Comment:
```suggestion
  # When updating libmongocrypt, consider also updating the copy of
  # libmongocrypt's kms-message in `src/kms-message`. Run
  # `.evergreen/scripts/kms-divergence-check.sh` to ensure that there is no
  # divergence in the copied files.
```

The dependent version of libmongocrypt seems like a separate concern from the version of kms-message vendored.

Example: if libmongoc requires new features in libmongocrypt, there may be no relevant changes in kms-message.

I suggest noting to consider updating the vendored copy of kms-message. This may help to remind. But requiring kms-message be copied when libmongocrypt is updated may not be strictly needed.